src/components/PullToRefresh/index.tsx

```diff
@@ -0,0 +1,63 @@
+import Router from 'next/router';
+import PR from 'pulltorefreshjs';
+import React, { useEffect } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { RefreshIcon } from '@heroicons/react/outline';
+import { defineMessages, useIntl } from 'react-intl';
+
+const messages = defineMessages({
+  reload: 'Reload',
+  reloading: 'Reloading',
+});
+
+const PullToRefresh: React.FC = () => {
+  const intl = useIntl();
+
+  useEffect(() => {
+    PR.init({
+      mainElement: '#pull-to-refresh',
+      onRefresh() {
+        Router.reload();
+      },
+      iconArrow: ReactDOMServer.renderToString(
+        <div className="z-50 flex items-center justify-center">
+          <RefreshIcon className="z-50 h-10 w-10 text-white" />
+        </div>
+      ),
+      iconRefreshing: ReactDOMServer.renderToString(
+        <div className="z-50 flex items-center justify-center">
+          <RefreshIcon className="z-50 h-10 w-10 animate-spin text-white" />
```

Comment:
```suggestion
          <RefreshIcon
            className="z-50 h-10 w-10 animate-spin text-white"
            style={{ animationDirection: 'reverse' }}
          />
```
So that the arrow doesn't spin backwards 😜

Reply:
Code updated. We are now spinning correctly :)